test/rekorsearchui/rekor_search_sign_verify_test.go

```diff
@@ -0,0 +1,388 @@
+package rekorsearchui
+
+import (
+	"encoding/json"
+	"fmt"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"strconv"
+	"strings"
+	"time"
+
+	"github.com/go-git/go-git/v5"
+	"github.com/go-git/go-git/v5/config"
+	"github.com/go-rod/rod"
+	"github.com/go-rod/rod/lib/launcher"
+	"github.com/google/uuid"
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+	"github.com/securesign/sigstore-e2e/pkg/api"
+	"github.com/securesign/sigstore-e2e/pkg/clients"
+	"github.com/securesign/sigstore-e2e/test/testsupport"
+	"github.com/sirupsen/logrus"
+	"github.com/ysmood/got"
+)
+
+var browser *rod.Browser
+var page *rod.Page
+
+type G struct {
+	Got     got.G
+	Browser *rod.Browser
+}
+
+func (g G) page(url string) *rod.Page {
+	page := g.Browser.MustPage(url)
+	DeferCleanup(func() {
+		if err := page.Close(); err != nil {
+			g.Got.Logf("Failed to close page: %v", err)
+		}
+	})
+	return page
+}
+
+var _ = Describe("Test the Rekor Search UI", Ordered, func() {
+	var (
+		err               error
+		rekorCli          *clients.RekorCli
+		gitsign           *clients.Gitsign
+		tempDir           string
+		dirFilePath       string
+		tarFilePath       string
+		config            *config.Config
+		signatureFilePath string
+		dir               string
+		repo              *git.Repository
+		testData          struct {
+			Email          string
+			Hash           string
+			LogIndex       int
+			EntryUUID      string
+			CommitSHA      string
+			CommitLogIndex int
+		}
+	)
+
+	appURL := api.GetValueFor(api.RekorUIURL)
+
+	setup := func() G {
+		launch := launcher.New().Headless(false) // if you want to see process of testing realtime, just rewrite true to false
+		url := launch.MustLaunch()
+		browser := rod.New().ControlURL(url).MustConnect()
+
+		g := G{Got: got.New(GinkgoT()), Browser: browser}
+		DeferCleanup(func() {
+			if err := browser.Close(); err != nil {
+				g.Got.Logf("Failed to close browser: %v", err)
+			}
+		})
+
+		return g
+	}
+
+	BeforeAll(func() {
+		err = testsupport.CheckMandatoryAPIConfigValues(api.OidcRealm)
```

Comment:
Add `api.RekorUIURL`

Reply:
@ssulka CheckMandatoryAPIConfigValues has variadic arguments so you can simply add as another argument `api.RekorUIURL`


```suggestion
		err = testsupport.CheckMandatoryAPIConfigValues(api.OidcRealm, api.RekorUIURL)
```